docs/source/involved-opendcs.rst

```diff
@@ -0,0 +1,78 @@
+###################
+OpenDCS Involvement
+###################
+
+********************************
+Sign Up for E-mail Notifications
+********************************
+
+Official Notifications - FreeLists
+==================================
+
+To sign up for the offical OpenDCS notifications subscribe
+to the FreeList e-mail. 
+
+https://www.freelists.org/list/opendcs 
+
+Only subscribers of this list serve can send e-mails. 
+
+.. image:: ./media/involved/opendcs/im-00-freelists-opendcs.JPG
+   :alt: subscribe free lists opendcs
+   :width: 550
+
+For an archive of old e-mails sent see the following link. 
+
+https://www.freelists.org/archive/opendcs/  
+
+.. image:: ./media/involved/opendcs/im-01-freelists-opendcs-archive.JPG
+   :alt: archive free lists opendcs
+   :width: 550
+
+
+Official Notifications for Developers- FreeLists
+================================================
+
+
+To sign up for the offical OpenDCS dev notifications subscribe
+to the FreeList e-mail. 
+
+https://www.freelists.org/list/opendcs-dev 
+
+Only subscribers of this list serve can send e-mails. 
+
+.. image:: ./media/involved/opendcs/im-02-freelists-opendcs-dev.JPG
+   :alt: subscribe free lists opendcs dev
+   :width: 550
+
+For an archive of old e-mails sent see the following link. 
+
+https://www.freelists.org/archive/opendcs-dev/  
+
+.. image:: ./media/involved/opendcs/im-03-freelists-opendcs-dev-archive.JPG
+   :alt: archive free lists opendcs dev
+   :width: 550
+
+
+**************************
+GitHub Repository Overview
+**************************
+
+
+opendcs
+=======
+
```

Comment:
This is the "Organization" URL, the URL for OpenDCS (the core project that most interact with) is https://github.com/opendcs/opendcs